melos.yaml

```diff
@@ -15,7 +16,7 @@ command:
       freezed_annotation: ^2.4.1
       http: ^1.1.0
       json_annotation: ^4.8.1
-      meta: ^1.11.0
+      meta: ^1.10.0
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 
Version had to be lowered due to compatibility issues.